README.md

```diff
@@ -137,14 +161,13 @@ Upscayl uses Real-ESRGAN-ncnn-vulkan binaries to upscale images. More models and
 # Donate
 
 <a href="https://www.buymeacoffee.com/fossisthefuture">
-<img src="https://user-images.githubusercontent.com/25067102/154570688-9e143f2b-fee3-4b05-a9d2-a7a3013b2b51.png" />
+  <img src="https://user-images.githubusercontent.com/25067102/154570688-9e143f2b-fee3-4b05-a9d2-a7a3013b2b51.png" />
 <a/>
 
 # Credits ❤
 
-- Real-ESRGAN for their wonderful research work.\
+- Real-ESRGAN for their wonderful research work\.  
 [Real-ESRGAN: Copyright (c) 2021, Xintao Wang](https://github.com/xinntao/Real-ESRGAN/)
-
```

Comment:
Is the backslash even needed now that the extra newline is gone?

Reply:
No it is not, you are right. Double space will do the job perfectly. [Done](https://github.com/Danik1601/upscayl/commit/b6b93377f2ea5d32ac3020e5fb6cbb4bfe683647)

Reply:
By the way, can we just insert the link into `Real-ESRGAN` and remove the whole `Real-ESRGAN: Copyright (c) 2021, Xintao Wang` line altogether?

Reply:
Maybe. You'll have to ask @NayamAmarshe though.

Reply:
I think we should give proper credits, the current one is fine :)